### ISSUE TITLE
when english?

### ISSUE THREAD

**Issue by piotrekkrol7** [CrazyHackGUT/SB_Material_Design#48]
Hi @CrazyHackGUT  when english version ?

**Comment by tallanvor** [comment]
@piotrekkrol7 Hi!
Multilang system freezed.
I just waiting for stable build 1.1.5...

**Comment by piotrekkrol7** [comment]
@tallanvor  yet we have stable not dev build 1.1.5 . When can we expect english version ? 💃

**Comment by pusty44** [comment]
Maybe some option to rewrite this bans for multilang with translations uploaded via xml? I want to use it, but it's in russian :(

**Comment by CrazyHackGUT** [comment]
@pusty44 @tallanvor work on this.

**Comment by tibehrius** [comment]
Please provide a multi-language mode. I can help with the PT-BR version if you want.

Congratulations on the excellent work!

**Comment by CrazyHackGUT** [comment]
@tibehrius we need multilanguage system for SB. @tallanvor was engaged in it, but nothing had been heard of him for 2 months.
I can try create my own implementation, but...I'm currently busy fixing some bugs in the SB, and i don't have time for this :(

**Comment by tallanvor** [comment]
I'm here..... Still working.. Very slowly....

2017-04-03 6:22 GMT+03:00 Kruzya <notifications@github.com>:

**Comment by CrazyHackGUT** [comment]
@tallanvor Nice. We're waiting.

**Comment by tibehrius** [comment]
Any news, sirs? s2

**Comment by tallanvor** [comment]
Today I have some free time. This work is so hard and fucking dirty
For 5 hours - about 50 lines just turns out.
Keep calm - it comes

2017-04-12 23:35 GMT+03:00 tibehrius <notifications@github.com>:

**Comment by tibehrius** [comment]
Pleaaase, in english/multi language ... :'(

**Comment by tibehrius** [comment]
Dear friends, please let me know if you need any help. I would very much like to use this mod = /

I have already installed the available version, but I can not understand anything.

https://bigcs.com.br/bans

Thanks!! :)

**Comment by tallanvor** [comment]
Duds, sorry, but my main job takes more than all my time....
Nobody can help my, without knowledge russian language.

**Comment by tibehrius** [comment]
I'm translating everything manually, directly into the source code of each file. I can already use it in my native language, and everything is working very well. The only problem is that when there is an update, I will lose all my work :(

Is not there a way to do this by following good practice methods, so that the translations will not be lost later?

Here is my version in PT-BR:

https://bigcs.com.br/bans/

Regards! :)

**Comment by CrazyHackGUT** [comment]
Looks nice!
But i don't understand nothing... :rofl:

**Comment by tibehrius** [comment]
Roftl!

Now you know how I felt when I first saw the panel :'D

Is it possible to use the "gettext" (PHP) extension to support multi-languages?

**Comment by tibehrius** [comment]
@tallanvor ... dear friend, did you give up doing the project? =(

**Comment by tallanvor** [comment]
@tibehrius, yes, I'm trying .... Catastrophically there is no time ... All you can just wait ... I do not like it either, but what can I do?
